### ISSUE TITLE
Better support for representing default values in introspection

### ISSUE THREAD

**Issue by mwildehahn** [mwildehahn/hack-graphql#103]
https://github.com/mwildehahn/hack-graphql/pull/102#discussion_r640070248

I started doing this but input objects are a little tricky so going to do this as a PR on it's own.

**Comment by jjergus** [comment]
> input objects are a little tricky

Yeah there might be a bit of a cyclical dependency there, you need the types from the InputValue shapes to output the default values, and you need the default values for the InputValue shapes :)
